packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceTimelineLink.test.js

```diff
@@ -13,30 +13,31 @@
 // limitations under the License.
 
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen, fireEvent, createEvent } from '@testing-library/react';
 
 import TraceTimelineLink from './TraceTimelineLink';
-import NewWindowIcon from '../../common/NewWindowIcon';
 
 describe('TraceTimelineLink', () => {
   const traceID = 'test-trace-id';
-  let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(<TraceTimelineLink traceID={traceID} />);
+    render(<TraceTimelineLink traceID={traceID} />);
   });
 
   it('renders the NewWindowIcon', () => {
-    expect(wrapper.find(NewWindowIcon).length).toBe(1);
+    expect(screen.getAllByTestId('NewWindowIcon').length).toBe(1);
   });
 
   it('links to the given trace', () => {
-    expect(wrapper.find('a').prop('href')).toBe(`/trace/${traceID}`);
+    expect(screen.getByRole('link').href).toBe(`http://localhost/trace/${traceID}`);
```

Comment:
interesting, why is this changing from no-host URL to `http://localhost`? That seems counterintuitive - localhost is very dependent on where the code runs

Reply:
cc @anshgoyalevil

Reply:
This is because AFAIK while rendering RTL actually converts all the the relative links to the actual complete URLs to which the same anchor is supposed to route to. I get the feeling of it being a bit counterintuitive, but since the tests would always be running in either locally or in workflows (and never in build or production), the same shouldn't be ambiguous or flaky.

Reply:
I didn't see any other tests asserting full URL with host name (unless the test provided the value like example.com), so I added URL parsing bit